client/src/endpoint/account.rs

```diff
@@ -452,3 +452,180 @@ mod effects_tests {
         assert_eq!(req.uri().query(), Some("cursor=CURSOR&order=asc&limit=123"));
     }
 }
+
+/// An endpoint that accesses the Payment operations where a specific account is either the
+/// sender or receiver.
+#[derive(Debug, Clone)]
+pub struct Payments {
+    id: String,
+    cursor: Option<String>,
+    order: Option<Order>,
+    limit: Option<u32>,
+}
+
+impl Payments {
+    /// Returns a new endpoint for account payments. Hand this to the client in order to request
+    /// payment operations for a specific account.
+    pub fn new(id: &str) -> Self {
+        Self {
+            id: id.to_string(),
+            cursor: None,
+            order: None,
+            limit: None,
+        }
+    }
+
+    /// Starts the page of results at a given cursor
+    ///
+    /// ## Example
+    ///
+    /// ```
+    /// use stellar_client::sync::Client;
```

Comment:
Since these tests are a bit involved and require a good deal of set up, let's just do it once in the struct documentation itself and just provide examples of setting the values in the function documentation.